docs/services/thorlabs_cube_motor_kinesis.rst

```diff
@@ -0,0 +1,54 @@
+Thorlabs Cube Motors
+====================
+
+This service connects to Thorlabs TDC001 and Thorlabs KDC101 controllers in order to operate a motor.
+

```

Comment:
It would be good to add the link to the Kinesis software:
 https://www.thorlabs.com/software_pages/viewsoftwarepage.cfm?code=Motion_Control#